featuretools/tests/primitive_tests/test_transform_features.py

```diff
@@ -116,35 +116,24 @@ def test_make_trans_feat(es):
 
 def test_diff(es):
     value = ft.Feature(es['log']['value'])
-    customer_id_feat = ft.Feature(es['sessions']['customer_id'], entity=es['log'])
-    diff1 = ft.Feature([value, es['log']['session_id']], primitive=Diff)
-    diff2 = ft.Feature([value, customer_id_feat], primitive=Diff)
+    diff1 = ft.Feature([value], primitive=Diff)
 
-    pandas_backend = PandasBackend(es, [diff1, diff2])
+    pandas_backend = PandasBackend(es, [diff1])
     df = pandas_backend.calculate_all_features(instance_ids=range(15),
                                                time_last=None)
 
     val1 = df[diff1.get_name()].values.tolist()
-    val2 = df[diff2.get_name()].values.tolist()
-    correct_vals1 = [
-        np.nan, 5, 5, 5, 5, np.nan, 1, 1, 1, np.nan, np.nan, 5, np.nan, 7, 7
-    ]
-    correct_vals2 = [np.nan, 5, 5, 5, 5, -20, 1, 1, 1, -3, np.nan, 5, -5, 7, 7]
+    correct_vals = [np.nan, 5, 5, 5, 5, -20, 1, 1, 1, -3, 0, 5, -5, 7, 7]
     for i, v in enumerate(val1):
         v1 = val1[i]
         if np.isnan(v1):
-            assert (np.isnan(correct_vals1[i]))
+            assert (np.isnan(correct_vals[i]))
         else:
-            assert v1 == correct_vals1[i]
-        v2 = val2[i]
-        if np.isnan(v2):
-            assert (np.isnan(correct_vals2[i]))
-        else:
-            assert v2 == correct_vals2[i]
+            assert v1 == correct_vals[i]
 
 
 def test_diff_single_value(es):
-    diff = ft.Feature([es['stores']['num_square_feet'], es['stores'][u'région_id']], primitive=Diff)
+    diff = ft.Feature([es['stores']['num_square_feet']], primitive=Diff)
```

Comment:
the groupby here is what tests the try/except we removed, we should keep it

Reply:
@rwedge how should I go about calling the groupby here? I tried just replacing primitive with groupby_trans_primitives, but that doesn't work. Do I need an extra line to add the groupby part? Or am I missing something about this comment?

Reply:
The [v0.7.0 release changelog](https://docs.featuretools.com/changelog.html) gives an example